<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>model</class>
 <widget class="QWidget" name="model">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1316</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>30</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;使用说明：&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑'; vertical-align:middle;&quot;&gt;1. 单击：选中点 &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑'; vertical-align:middle;&quot;&gt;2. 双击右键：增加顶点 &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑'; vertical-align:middle;&quot;&gt;3. 长按左键：拖动点 &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑'; vertical-align:middle;&quot;&gt;4. 双击左键：删除顶点&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="isEditModel">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>170</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑模型</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveModel">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>460</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>保存模型</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadModel">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>520</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>加载模型</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cleanbtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>630</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>清空画板</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setCenter">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>220</y>
     <width>93</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>设置中心</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectImg">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>310</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>选择图片</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
